README.md

```diff
@@ -5,3 +5,85 @@
+
+## Contents
+
+1. [Development](#development)
+1. [Input Data Format](#input-data-format)
+
+## Development
+
+### Getting set up
+
+1. Grab the source:
+
+   `git clone git@github.com:Recidiviz/justice-counts-components.git`
+
+1. Install Yarn package manager:
+
+   `brew install yarn`
+
+   For alternative Yarn installation options, see [Yarn Installation](https://yarnpkg.com/en/docs/install).
+
+1. Install dependencies:
+
+   `yarn install`
+
+That's it! We suggest installing a linting package for your preferred code editor that hooks into [eslint](#eslint), such as [linter-eslint](https://atom.io/packages/linter-eslint) for Atom.
+
+### Testing and linting
+
+To run tests wired up through react-scripts:
+
+`yarn test --coverage`
+
+Running tests this way will also write code coverage statistics to stdout and the `coverage` directory, which are reported to Coveralls.
+
+To run linting:
+
+`yarn lint`
+
+Linting is done with a combination of `eslint` and `Prettier`. `react-scripts` runs some minimal linting by default as part of its build process; errors on those linting rules would cause build scripts to fail, but errors in the broader [configuration](https://github.com/Recidiviz/supervision-success-component/.eslintrc.json) should not.
+
+To have eslint and Prettier fix violations automatically, run:
+
+`yarn lint --fix`
+
+### Running the app locally
+
+A yarn script is available for starting the development server. The entire app is a React frontend that is served out of port `3000`. This will also automatically open a browser to localhost on the appropriate port, pointing to the frontend.
+
+`yarn start`
+
+The development server will remain active until you either close your terminal or shut down the entire setup at once using `CTRL+c`.
+
+### Creating a build
+
+To generate a build of the frontend that is suitable for deploys, run: `yarn build`.
+
+Each time this is run, the `/build` directory will be wiped clean.
+
+## Input Data Format
+
+App should receive 2 required props: `data` and `stateCode`.
+
+`stateCode` controls state for which data should be visualized.
+
+`data` is the array of metric data, format could be found below.
+
+Now the `data.json` file is used to display data. It lives in `src/data.json` but could be moved around trivially, or pulled from a CDN or an API, for different deployment styles.
+
+An _example_ of a valid `data.json` file can be found in [`data.json`](src/data.json).
+
+`data.json` is the array that consists of JSON objects with fields:
+
+- `state_code` - state code e.g. `US_CO`
+- `metric` - metric name e.g. `RELEASES`
+- `year` - year e.g. `2020`
+- `month` - month (01-12) e.g. `02`
+- `date_reported` - Date, when metric was reported. e.g. `2019-12-31`
+- `value` - Value of the metric. e.g. `500`
+- `compared_to_year` - Year, which is used for comparing. e.g. `2019`
+- `compared_to_month` - Month, which is used for comparing. e.g. `04`
```

Comment:
Language tweak: "The month of the comparison data point, which is the most recent data point to compare against for trend analysis, no less than one year before this current data point, e.g. `04`"